File: app/src/main/java/io/twere/plainpure/data/model/dribbble/realm/FolloweeRealm.java
package io.twere.plainpure.data.model.dribbble.realm;

import io.realm.RealmModel;
import io.realm.annotations.PrimaryKey;
import io.realm.annotations.RealmClass;
import java.util.Date;

@RealmClass public class FolloweeRealm implements RealmModel {

  @PrimaryKey private long id;

  private Date createdAt;

  private UserRealm user;

  public long getId() {
    return id;
  }

  public void setId(long id) {
    this.id = id;
  }

  public Date getCreatedAt() {
    return createdAt;
  }

  public void setCreatedAt(Date createdAt) {
    this.createdAt = createdAt;
  }

  public UserRealm getUser() {
    return user;
  }

  public void setUser(UserRealm user) {
    this.user = user;
  }
}

File: app/src/main/kotlin/io/twere/plainpure/data/local/followee/FolloweeRealm.kt
package io.twere.plainpure.data.local.followee

import io.twere.plainpure.data.local.user.UserRealm
import io.realm.RealmModel
import io.realm.annotations.PrimaryKey
import io.realm.annotations.RealmClass
import java.util.Date

@RealmClass
open class FolloweeRealm : RealmModel {

  @PrimaryKey
  open var id: Long = 0
  open var createdAt: Date? = null
  open var followee: UserRealm? = null

}